sections/futures/Trade/DelayedOrderConfirmationModal.tsx

```diff
@@ -155,6 +164,11 @@ const DelayedOrderConfirmationModal: FC = () => {
 		]
 	);
 
+	const truncateLeverage = useMemo(
+		() => truncateNumbers(wei(Number(leverageInput) ?? 0), DEFAULT_FIAT_DECIMALS),
```

Comment:
We'll probably want to use the calculated leverage in `potentialTradeDetails` here